cmd/dataset/main.go

```diff
@@ -75,6 +79,8 @@ func init() {
 	_ = datav1alpha1.AddToScheme(scheme)
 
 	startCmd.Flags().StringVarP(&metricsAddr, "metrics-addr", "", ":8080", "The address the metric endpoint binds to.")
+	startCmd.Flags().StringVarP(&statisticsAddr, "statistics-addr", "", ":6060", "The address the application runtime statistics endpoint binds to.")
+	startCmd.Flags().BoolVarP(&enablePerformanceAnalysis, "enable-performance-analysis", "", false, "Enable performance analysis by instant live visualization of application runtime statistics")
```

Comment:
@yangyuliufeng Thanks very much for your review. I will fix all. 
By the way, which is better between enable-performance-analysis and enable-statistics-visualization here?